.github/workflows/ci.yml

```diff
@@ -145,7 +146,7 @@ jobs:
         uses: actions/cache@v3
         with:
           path: .venv
-          key: venv-${{ runner.os }}-${{ hashFiles('**/poetry.lock') }}
+          key: venv-${{ runner.os }}-${{ steps.setup-python.outputs.python-version }}-${{ hashFiles('**/pyproject.toml') }}
```

Comment:
1. Using `${{ steps.setup-python.outputs.python-version }}` because `${{ matrix.pyver }}` resolves to nothing. 🤷‍♂️ 
2. Using the hash of the `pyproject.toml` because `poetry.lock` got removed in #457.

Reply:
${{ matrix.pyver }} perhaps

Reply:
Hmm... 🤔 

Last time I tried 2 weeks ago, `${{ matrix.pyver }}` did not work (resolved to nothing). That's why I mentioned it in the comments.

But today it worked. 🤷‍♂️ 

Maybe GitHub Action got update, or maybe I mistyped something...

So I fixed the code to use `matrix.pyver`.